### DISCUSSION TITLE
Welcome to PaperPi Discussions!

### DISCUSSION THREAD

**Discussion by txoof** [txoof/PaperPi#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by txoof** [comment]
@aaronr8684 
For a more realtime experience there's also [Discord](https://discord.com/channels/785572711149600799/785572711584890920)

I'm in +1 GMT (Amsterdam/Berlin); 17:00-22:00 are my prime hacking times.

**Reply by aaronr8684** [reply]
Yea on there as well (FlyinRiz#8684).

I'm -5 GMT (EST) and if I'm having a slow day, I put most of this kind of work in during my 9-5 job, but I also tend to work in the evenings a few nights a week 8p-12a

**Reply by txoof** [reply]
I'm txoof#0376 -- It looks like you're set to not accept anon friend requests.

I don't use discord all that often and kinda forgot how it works :)

**Reply by aaronr8684** [reply]
Yea, I mod a few servers so I turned it off because I was getting so much spam requests. Forgot about that.

**Reply by aaronr8684** [reply]
Sent a request. Do you have a server set up for the EPD stuff?
